<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Incolata</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::UserAvailable"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="headerLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLCDNumber" name="rawCountLCD">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>#/cmÂ³</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="mainAreaSplitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="leftSectionWidget">
       <layout class="QVBoxLayout" name="leftSection">
        <property name="rightMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QTableView" name="testTable">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>2</verstretch>
           </sizepolicy>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="FitFactorChartView" name="ffGraph">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>250</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="layout_controlbox">
           <item row="1" column="2" colspan="2">
            <widget class="QPushButton" name="startTest2">
             <property name="text">
              <string>Crash 2.5</string>
             </property>
             <property name="autoDefault">
              <bool>true</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QPushButton" name="startTest1">
             <property name="text">
              <string>OSHA</string>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="4" colspan="2">
            <widget class="QPushButton" name="startTest3">
             <property name="text">
              <string>OSHA mod (fast) FFP</string>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="3">
            <widget class="QComboBox" name="specimenSelector">
             <property name="editable">
              <bool>true</bool>
             </property>
             <property name="currentText">
              <string>Select Specimen</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3" colspan="3">
            <widget class="QComboBox" name="subjectSelector">
             <property name="editable">
              <bool>true</bool>
             </property>
             <property name="currentText">
              <string>Subject</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="graphLayoutWidget">
       <layout class="QGridLayout" name="graphLayout">
        <item row="1" column="1">
         <widget class="SequentialSeriesScrollingChartView" name="ambientSampleGraph">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="SequentialSeriesScrollingChartView" name="rawChartView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>250</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="SequentialSeriesScrollingChartView" name="specimenSampleGraph">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="SequentialSeriesScrollingChartView" name="liveFFGraph">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>250</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SequentialSeriesScrollingChartView</class>
   <extends>QGraphicsView</extends>
   <header>sequentialseriesscrollingchartview.h</header>
  </customwidget>
  <customwidget>
   <class>FitFactorChartView</class>
   <extends>QGraphicsView</extends>
   <header>fitfactorchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
